src/relay/WalletSDKRelay.ts

```diff
@@ -168,7 +169,6 @@ export class WalletSDKRelay extends WalletSDKRelayAbstract {
         .subscribe()
     );
 
-    // if session is marked destroyed, reset and reload
```

Comment:
i opted to remove this instead of updating it to reflect the new behavior of reset & reload when not in headless mode, and only reset when in headless mode. felt like that was too verbose of a code comment